File: C Language/Assignment/Assign 11/palindrome.c
#include<stdio.h>
#include<string.h>
void main()
{
    char str[30];
    int i,len=0,flag=1;
    printf("enter a string:");
    scanf("%s",str);


   for(i=0;str[i]!='\0';i++)
   {
    if(str[i]=='\n')break;
    len++;
   }

   for(i=0;i<len/2;i++)
   {
    if(str[i]!=str[len-1-i])
    flag=0;
    break;
   }

   if(flag==1)
   printf("palindrome");
   else
   printf("not palindrome");
}

File: Java/Assignments/Assign 1/Palindrome.java
package assign1;
import java.util.Scanner;
public class Palindrome {

	public static void main(String[] args) {
		int a,no,rem,rev=0;
		 Scanner sc = new Scanner(System.in);  
	        System.out.print("Enter the number: ");
	        a = sc.nextInt();
	    int temp=a;
	    no=a;

	    while (temp!=0)
	    {
	        rem=temp%10;
	        rev=rev*10+rem;
	        temp=temp/10;
	    }
	    if(no==rev){
	        System.out.printf("number is palindrome");
	    }else{
	    	System.out.printf("number is not palindrome");
	    }
	    sc.close();

	}

}
